components/notifications.js

```diff
@@ -528,11 +529,27 @@ function WithdrawlPaid ({ n }) {
 }
 
 function Referral ({ n }) {
+  const { me } = useMe()
+  let referralSource = 'of you'
+  switch (n.source?.__typename) {
+    case 'Item':
+      referralSource = (Number(me?.id) === Number(n.source.user?.id) ? 'of your' : 'you shared this') + ' ' + (n.source.title ? 'post' : 'comment')
+      break
+    case 'Sub':
+      referralSource = (Number(me?.id) === Number(n.source.userId) ? 'of your' : 'you shared the') + ' ~' + n.source.name + ' territory'
+      break
+    case 'User':
+      referralSource = (me?.name === n.source.name ? 'of your profile' : `you shared ${n.source.name}'s profile`)
+      break
+  }
```

Comment:
I mostly just made the copy a little more natural sounding if the source was your own item/territory/profile.